js/src/immersive-reader-sdk.ts

```diff
@@ -5,7 +5,18 @@ import { renderButtons } from './renderButtons';
 import { close, launchAsync } from './launchAsync';
 
 window.addEventListener('load', () => {
+    if (!('Promise' in window)) {
+        dynamicallyLoadScript('https://contentstorage.onenote.office.net/onenoteltir/permanent-static-resources/promise-polyfill.min.js');
+    }
+
     renderButtons();
 });
 
-export {renderButtons, close, launchAsync};
\ No newline at end of file
+function dynamicallyLoadScript(scriptUrl: string) {
+    const script = document.createElement('script');
+    script.src = scriptUrl;
+    script.async = true;
```

Comment:
comment to explain why async?

Reply:
I removed the line since I found out dynamically loaded scripts are async by default.